scripts/update-image.py

```diff
@@ -9,21 +9,21 @@
 global_mount_info = None
 sfdisk_command = None
 
-@attr.s
 class MountInfo:
-	blockdev = attr.ib()
-	mount_using = attr.ib()
-	image = attr.ib()
-	mountpoint = attr.ib()
-	root_idx = attr.ib()
-	efi_idx = attr.ib()
-	root_uuid = attr.ib()
-
-@attr.s
+	def __init__(self, blockdev, mount_using, image, mountpoint, root_idx, efi_idx, root_uuid):
+		self.blockdev = blockdev
+		self.mount_using = mount_using
+		self.image = image
+		self.mountpoint = mountpoint
+		self.root_idx = root_idx
+		self.efi_idx = efi_idx
+		self.root_uuid = root_uuid
+
```

Comment:
No, attr.ib is used for a reason.

Reply:
I get that. But AFAICT the only feature we use is __init__. So I've just removed attrs entirely from this script.